backend/pkg/database/apiinventory.go

```diff
@@ -54,6 +54,8 @@ type APIInfo struct {
 	ReconstructedSpecInfo string         `json:"reconstructedSpecInfo,omitempty" gorm:"column:reconstructed_spec_info" faker:"-"`
 	ProvidedSpec          string         `json:"providedSpec,omitempty" gorm:"column:provided_spec" faker:"-"`
 	ProvidedSpecInfo      string         `json:"providedSpecInfo,omitempty" gorm:"column:provided_spec_info" faker:"-"`
+
+	Annotations []*APIInfoAnnotation `gorm:"foreignKey:APIID;references:ID"`
```

Comment:
where it is used?

Reply:
It is not used anywhere, I added it just to signal that it is a foreign key, should I delete it?

Reply:
not sure I understand, if it not used in the system what does it mean it's a foreign key?

Reply:
we have a table for inventory items annotations called api_annotations, that has a one to many correlation with api_inventory. Usually we use api_annotations table independently of api_inventory, but I still want to empathise that there is such relation between these two tables.